src/Components/Components/ref/Microsoft.AspNetCore.Components.netcoreapp.cs

```diff
@@ -403,6 +403,7 @@ public sealed partial class RenderTreeBuilder : System.IDisposable
     {
         public RenderTreeBuilder() { }
         public void AddAttribute(int sequence, in Microsoft.AspNetCore.Components.RenderTree.RenderTreeFrame frame) { }
+        public void AddAttribute(int sequence, string name) { }
```

Comment:
Is this a perf optimization?

Reply:
I think I answered my own question [here](https://github.com/dotnet/aspnetcore/pull/23703/files#r450629775). You can correct me if I'm wrong.

Reply:
Yes: https://github.com/dotnet/aspnetcore/pull/23703#issuecomment-654324052